gemrb/core/System/VFS.cpp

```diff
@@ -320,6 +327,9 @@ void FixPath (char *path, bool needslash)
 
 	if (needslash) {
 		if (path[i] == PathDelimiter) return;
+#ifdef __amigaos4__
+                if (path[i] == ':') return;
```

Comment:
I dont understand the need for this. if `:` is the PathDelimiter on amiga just set it that way

Reply:
Because AmigaOS path works in this way:

DH0:Dir1/Dir2/Dir3

Where DH0 is the device (and could be any name ending in : )  and then there are the directories separated by /

Reply:
hmmm, isn't that how windows works too?

Reply:
Honestly i don't understand how can work on windows. There is no ":" in the code. Most probably because the PathDelimiter is \.
I've tried and without these changes the game won't start because gemrb doesn't find the files